### PR TITLE
Moving CSS to modules

### PR DESCRIPTION
**Pull request by stubelshe**
Moving elements to the component level and out of global.

### CODE REVIEW

File: src/components/Transactions/Transaction.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Transaction = ({
       <div className="level-item">
         <p className="subtitle is-6">
           {amount}
-          <span className={`icon-transfer${kind === "deposit" ? "_to" : "_from"} u-ml1`}></span>
+          <span className={cx(kind === "deposit" ?  `icon-transfer_to ${css.iconTransferTo}` : `icon-transfer_from ${css.iconTransferFrom}` , "u-ml1")}></span>
```
Comment (ceanela) [inline src/components/Transactions/Transaction.js]: Glad this worked out for you!

File: src/components/Footer/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "../../styles/custom_variables.scss";
+
+footer {
+	border-top: 1px solid $grey-mid;
```
Comment (ceanela) [inline src/components/Footer/styles.module.scss]: S   p   a    c    i     n   g       looks a bit  off here.

Comment (stubelshe) [reply src/components/Footer/styles.module.scss]: Ahh, yeah it moved to 4 spaces when i copied and pasted, I'll fix it.

File: src/components/Footer/styles.module.scss [hunk 3]
```diff
@@ -7,15 +7,15 @@ footer {
 	font-size: 0.875rem;
```
Comment (stubelshe) [inline src/components/Footer/styles.module.scss]: Spacing should be resolved.

**Comment by ceanela** [comment]
Fixed your conflicts

**Pull request merged**